includes/tracks/class-wc-tracks-client.php

```diff
@@ -92,10 +92,49 @@ public static function build_timestamp() {
 		return number_format( $ts, 0, '', '' );
 	}
 
+	/**
+	 * Get a user's identity to send to Tracks. If Jetpack exists, default to its implementation.
+	 *
+	 * @param int $user_id User id.
+	 * @return array Identity properties.
+	 */
+	public static function get_identity( $user_id ) {
+		if ( class_exists( 'Jetpack' ) ) {
```

Comment:
I think I pointed out in another PR, this might not be enough to know if Jetpack is really there. Can we add a file exists on jetpack/_inc/lib/tracks/client.php to be sure before trying to include it?